### ISSUE TITLE
square_error_cost

### ISSUE THREAD

**Issue by m-gopichand** [unifyai/ivy#17425]

**Comment by morady0213** [comment]
loss = MSELoss()
for epoch in range(num_epochs):
    for batch in data_loader:
        inputs, targets = batch
        outputs = model(inputs)
        loss = loss(outputs, targets)
        loss.backward()
        optimizer.step()
        optimizer.clear_grad()
        print(f"Epoch {epoch+1}, Loss: {loss.numpy()}")

**Comment by m-gopichand** [comment]
Hi, @hmahmood24 can you review this.

**Comment by Ambro19** [comment]
Hi @m-gopichand , can I try to work on this issue: https://github.com/unifyai/ivy/issues/15053#issue-1700247356
